flake.nix

```diff
@@ -101,13 +102,16 @@
       lenovo-thinkpad-t590 = import ./lenovo/thinkpad/t590;
       lenovo-thinkpad-x1 = import ./lenovo/thinkpad/x1;
       lenovo-thinkpad-x1-yoga = import ./lenovo/thinkpad/x1/yoga;
-      lenovo-thinkpad-x1-yoga-7th-gen = import ./lenovo/thinkpad/x1/yoga/7th-gen;
+      lenovo-thinkpad-x1-yoga-7th-gen =
+        import ./lenovo/thinkpad/x1/yoga/7th-gen;
```

Comment:
Please avoid reformatting of unrelated profiles. If we want to change the style it should better done by enforcing a code formatter test in CI.

Reply:
oops, that was my IDE automatically, will revert it out of the diff